OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -839,9 +748,7 @@ void FindNewUnits(Actor self)
 
 			foreach (var a in newUnits)
 			{
-				if (a.Info.HasTraitInfo<HarvesterInfo>())
-					QueueOrder(new Order("Harvest", a, false));
-				else
+				if (!a.Info.HasTraitInfo<HarvesterInfo>())
```

Comment:
Can we add harvesters to `Info.UnitsCommonNames.ExcludeFromSquads` in the yaml and then drop this explicit check?

Reply:
@reaperrr  consider a :+1: from me when this nit is resolved.